tools/cross-toolchain/README.md

```diff
@@ -0,0 +1,40 @@
+
+
+### Cross compile target support
+
+| target  | linux_amd64 | linux_arm64 | osx_amd64 | windows_amd64 |
+|----------|-------------------|------------------|-----------------|-----------------------|
+| `//beacon-chain` | :heavy_check_mark:  docker-sandbox and RBE, libkafka supported locally only | :heavy_check_mark:  docker-sandbox and RBE, no libkafka support | :heavy_check_mark:  docker-sandbox, no libkafka support | :heavy_check_mark:  docker-sandbox, no libkafka support |
+| `//validator`| :heavy_check_mark:  docker-sandbox and RBE | :heavy_check_mark: docker-sandbox and RBE | :heavy_check_mark:  docker-sandbox | :x:  Doesn't work. [#5008](https://github.com/prysmaticlabs/prysm/issues/5008) |
+
+The configurations above are enforced via pull request presubmit checks.
+
+### Bazel config flag values
+
+Use these values with `--config=<flag>`, multiple times if more than one value is defined in the table. Example: `bazel build //beacon-chain --config=windows_amd64_docker` to build windows binary in a docker sandbox.
+
+| Config | linux_amd64 | linux_arm64 | osx_amd64 | windows_amd64 |
+|----------|-------------------|------------------|-----------------|-----------------------|
+| Local run | `linux_amd64` | `linux_arm64` | `osx_amd64` | `windows_amd64` | 
+| Docker sandbox | `linux_amd64_docker` | `linux_arm64_docker` | `osx_amd64_docker` | `windows_amd64_docker `|
+| RBE (See [Caveats](#caveats)) | `linux_amd64` and `remote` | `linux_arm64`  and `remote` | `osx_amd64`  and `remote` | `windows_amd64`  and `remote` |
+
+### Caveats
+
+There are a few caveats to each of these strategies.
+
+- Local runs require clang compiler and the appropriate cross compilers installed. These runs should only be considered for a power user or user with specific build requirements. See the Dockerfile setup scripts to understand what dependencies must be installed and where.
+- Docker sandbox is *slow*. Like really slow! The purpose of the docker sandbox is to test RBE builds without deploying a full RBE system. Each build action is executed in its own container. Given the large number of small targets in this project, the overhead of creating docker containers makes this strategy the slowest of all, but requires zero additional setup.
+- Remote Build Execution is by far the fastest, if you have a RBE backend available. This is another advanced use case which will require two config flags above as well as additional flags to specify the `--remote_executor`. Some of these flags are present in the project `.bazelrc` with example values, but commented out.
+- Building with libkafka (`--define kafka_enabled=true`) is not supported with docker-sandbox or RBE at this time. Likely due to missing cmake dependencies in the builder image or lack of configuration via toolchains. 
```

Comment:
If there's an existing issue number, feel free to add it here

Reply:
There is not one yet. It's not a very widely used feature yet.